app/models/graphql/content_item.rb

```diff
@@ -0,0 +1,11 @@
+class Graphql::ContentItem < ContentItem
+  def self.find!(base_path)
+    content_item = Services.publishing_api.graphql_live_content_item(base_path)
+    content_item_hash = content_item.to_h
+    content_item_hash["cache_control"] = {
+      "max_age" => content_item.cache_control["max-age"],
+      "public" => !content_item.cache_control.private?,
+    }
```

Comment:
I'm curious why this is necessary

Reply:
It appears to be possible to override the maximum caching time in content items, through the `max_age`.

Although I [haven't added support](https://github.com/alphagov/publishing-api/blob/5f39effbfff806cce6da5580ee2a402c9c13ac28/app/controllers/graphql_controller.rb#L157) for this to Publishing API yet as none of the content types we've migrated use this, so probably don't need it here yet.  But decided to leave it in, so it's not forgotten when we add additional content types and to remain consistent with the equivalent code for Content Store.

Similar reasons for leaving in the `public` key, even though we're not yet serving drafts.

Reply:
The same happens for Content Store too:
https://github.com/alphagov/collections/blob/8342d039e0a94ba07eede6b3f70f6e2b3f6b7929/app/models/content_item.rb#L7-L10

I'm not quite sure why Collections works like this, but the [Content Store side of things](https://github.com/alphagov/content-store/blob/029bcc31d5f1efcb2696b637acfd80b1fc98026b/app/controllers/content_items_controller.rb#L159-L163) either uses default values or (if set) the `max_cache_time` from the content item to set this header.